manim/utils/tex_file_writing.py

```diff
@@ -126,8 +126,8 @@ def tex_compilation_command(tex_compiler, output_format, tex_file, tex_dir):
             outflag,
             "-interaction=batchmode",
             "-halt-on-error",
-            '-output-directory="{}"'.format(tex_dir),
-            '"{}"'.format(tex_file),
+            f'-output-directory="{tex_dir}"',
+            f'"{tex_file}"',
```

Comment:
Why is here the order  `'"{tex_file}"'` and not  `"'{tex_file}'"`

Reply:
Same as above: I wanted to stick to the format used before. Probably not a good idea in hindsight.

Reply:
> Oh, also: if you don't provide a separate one-line summary, I'll simply use the PR title for the corresponding changelog entry.

Actually, I did not consider the change significant enough to be worth an entry to the changelog....

Reply:
Alright!